### DISCUSSION TITLE
Lint를 활용했을 때의 장점에 대해 토의해봅시다!

### DISCUSSION THREAD

**Discussion by Hansolkkim** [Damagucci-Juice/sidedish#10] [General]
리뷰어님께서 SwiftLint를 활용헀을 때 어떤 점이 좋아졌는지 여쭤보셨는데, 여기에 대해서 얘기나눠봐요!

**Comment by Hansolkkim** [comment]
저는 여러 collaborator들끼리 같은 제약 조건을 가지고 코드를 짤 수 있어서, 상대방의 코드를 좀 더 이해하기 쉽지 않을까 생각했습니다!

**Reply by Damagucci-Juice** [reply]
맞습니다. 하지만 노란 경고창이 많아지는 단점도 있네요 ㅠㅠ 뭐랄까 기본으로 xcode에서 자동완성 시켜주는 긴 메서드들은 저희가 어떻게 쪼개고 나눠야 가독성이 좋아질지 고민해야하는 부분인것같습니다.

**Reply by Hansolkkim** [reply]
아, 제가 그 부분 한 번 건드려봤는데, 매개변수 몇 개 마다 줄 띄워주니까 노란 경고창이 사라졌습니다! 한 번 같이 확인해보고 노란 경고창 줄여봐요!!
